tests/unit/test_live_governance_contract.py

```
@@ -0,0 +1,328 @@
+def test_run_artifact_manifest_must_use_the_declared_artifact_sink() -> None:
+    contract = LiveGovernanceContract.model_validate(_contract_payload())
+    manifest_payload = _run_manifest_payload()
+    manifest_payload["artifact_root"] = "outputs/runtime/run-001"
+
+    with pytest.raises(ValueError, match="artifact sink root"):
+        contract.validate_run_artifact_manifest(manifest_payload)
+
+
+@pytest.mark.parametrize(
+    "artifact_path",
+    ["/tmp/proof.json", "../proof.json", "C:proof.json", "."],
```

Ruff (0.16.0)

[error] Probable insecure usage of temporary file or directory: "/tmp/proof.json"

(S108)